packages/builder/src/components/automation/AutomationBuilder/FlowChart/ActionModal.svelte

```diff
@@ -0,0 +1,149 @@
+<script>
+  import { ModalContent, Layout, Detail, Body, Icon } from "@budibase/bbui"
+  import { automationStore } from "builderStore"
+  import DiscordLogo from "assets/discord.svg"
+  import ZapierLogo from "assets/zapier.png"
+  import IntegromatLogo from "assets/integromat.png"
+  import SlackLogo from "assets/slack.svg"
+  import n8nlogo from "assets/n8n.png"
+
+  import { database } from "stores/backend"
+  $: instanceId = $database._id
+
+  let selectedAction
+  let actionVal
+  export let blockComplete
```

Comment:
NAB: We should probably have a lint rule or something for this, but generally the order of the svelte files are:

```
const atTheTop

export let propsAfterThat

let thenVariables

$: reactiveStuff

function utilityFuncsAtTheEnd() {}
```